test/addons/dlopen-ping-pong/test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const common = require('../../common');
+
+if (common.isWindows) {
+  common.skip('dlopen global symbol loading is not supported on this os.');
+  return;
+}
+
+const assert = require('assert');
+const path = require('path');
+const os = require('os');
+
+const bindingPath = require.resolve(`./build/${common.buildType}/binding`);
+process.dlopen(module, bindingPath,
+               os.constants.dlopen.RTLD_NOW | os.constants.dlopen.RTLD_GLOBAL);
+module.exports.load(path.dirname(bindingPath) + '/ping.so');
+assert.strictEqual(module.exports.ping(), 'pong');
+console.log('binding.ping() =', module.exports.ping());
```

Comment:
The value is purely completeness of the test. It is entirely feasible that a user (or some set of dependencies) to do this kind of thing and it would be helpful to have verification of the expected behavior.